custom_components/blitzortung/translations/sk.json

```diff
@@ -0,0 +1,65 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "Nastavte detekciu bleskov Blitzortung",
+        "data": {
+          "name": "Názov inštancie integrácie",
+          "latitude": "Zem. šírka",
+          "longitude": "Zem. dĺžka"
```

Comment:
Please do not use abbreviations, core integration translations do not use abbreviations here

![obraz](https://github.com/user-attachments/assets/cce22eda-b12d-4f55-85ce-9a90a79b291d)